libs/ledger-live-common/src/families/bitcoin/wallet-btc/pickingstrategies/Merge.ts

```diff
@@ -8,6 +8,9 @@ import * as utils from "../utils";
 import { log } from "@ledgerhq/logs";
 import { OutputInfo } from "..";
 
+/**
+ * prioritizes the smallest UTXOs
```

Comment:
What means "smallest" here ?

Reply:
it means "prioritize using UTXOs with smaller denominations"
I've changed the wording now to be more accurate.